cpp/find_missing_references.cc

```diff
@@ -108,10 +108,10 @@ int Main(int argc, char *argv[]) {
         archive_writer.write(missing_references_text);
         archive_writer.close();
 
-        const auto status_code(EmailSender::SendEmail("nobody@nowhere.com", email_address, "Missing PPN's",
-                                                      "Attached is the new list of " + std::to_string(new_missing_ppns.size()) + " missing PPN('s).",
-                                                      EmailSender::DO_NOT_SET_PRIORITY, EmailSender::PLAIN_TEXT, /* reply_to = */"",
-                                                      /* use_ssl = */true, /* use_authentication = */true, { ZIP_FILENAME }));
+        const auto status_code(EmailSender::SendEmailWithFileAttachments(
+            "nobody@nowhere.com", { email_address }, "Missing PPN's",
```

Comment:
This should also be changed to `no_reply@ub.uni-tuebingen.de` when time permits

Reply:
How about setting and re-using a constant for this? EmailSender::DEFAULT_EMAIL_SENDER or something